autocodesign/devportalclient/appstoreconnectclient/profiles.go

```diff
@@ -342,11 +342,9 @@ func (c *ProfileClient) SyncBundleID(bundleID appstoreconnect.BundleID, appEntit
 }
 
 func wrapInProfileError(err error) error {
-	if respErr, ok := err.(appstoreconnect.ErrorResponse); ok {
```

Comment:
NOTE: should not we use errors.Is or errors.As instead of the type cast?

Reply:
updated

Reply:
Just a question about this piece of code because you wrote `updated` but the type casting is still there. Is it on purpose?

Reply:
It was a mistake, my bad.